### DISCUSSION TITLE
Widget to launch a LUA script?

### DISCUSSION THREAD

**Discussion by Jonathan-Ott** [EdgeTX/edgetx#2404] [Ideas]
(For radios like the Radiomaster TX16S)

Is there a widget to launch a LUA script, like to launch the Betaflight LUA, without having to go into 'Tools' then 'Betaflight Setup'? |

Just curious (or maybe to plant a seed in the minds of programmers wondering what to make next.)

Thanks!

Jon

**Comment by pfeerick** [comment]
Is there a built-widget - no. But it should be easy enough for some one to make a simple Lua widget ;)

**Reply by Jonathan-Ott** [reply]
That settles it. We need a Betaflight LUA Widget!

I'd do it, but I'm happy if I can turn the radio on. ;)

**Reply by pfeerick** [reply]
Dang it! I want it too, and was hoping you'd say you were on the case... 😆 I'm sure we'll get that soon... if not I might finally do something about it.

**Comment by raphaelcoeffic** [comment]
> Is there a widget to launch a LUA script, like to launch the Betaflight LUA, without having to go into 'Tools' then 'Betaflight Setup'?

Aha, so basically a shortcut Widget? That is something that we should think about while moving to a more "App-like" feeling implementation.

**Reply by Jonathan-Ott** [reply]
@pfeerick @raphaelcoeffic 

This [YouTube video](https://www.youtube.com/watch?v=zZq9JpHpGnk) is what I did on the QX7 (not my video) to get to the Betaflight script easier. Is this something we could incorporate into EdgeTX? 

Thanks!

Jon

**Reply by raphaelcoeffic** [reply]
The BF standalone script then needs to converted/adapted to be a widget, and can implement different things based on whether or not it's running in fullscreen-mode.

**Comment by nhut6789** [comment]
I like this too!
